### ISSUE TITLE
How to use training datasets in multiple folders in 1.x branch

### ISSUE THREAD

**Issue by zsc1220** [open-mmlab/mmsegmentation#2637]
In 0.x branch, can use a list to add training datasets in multiple folders, But in 1.x branch, where i should use a list ?

**Comment by zsc1220** [comment]
@MeowZheng Could you help me, thank you so much !

**Comment by zsc1220** [comment]
i get it , like this

`dataset_train = dict(
    type=dataset_type,
    data_root=data_root,
    data_prefix=dict(img_path='JPEGImages', seg_map_path='SegmentationClass'),
    ann_file='ImageSets/Segmentation/train.txt',
    pipeline=train_pipeline)

dataset_aug = dict(
    type=dataset_type,
    data_root=data_root,
    data_prefix=dict(
        img_path='JPEGImages', seg_map_path='SegmentationClassAug'),
    ann_file='ImageSets/Segmentation/aug.txt',
    pipeline=train_pipeline)

train_dataloader = dict(
    batch_size=4,
    num_workers=4,
    persistent_workers=True,
    sampler=dict(type='InfiniteSampler', shuffle=True),
    dataset=dict(type='ConcatDataset', datasets=[dataset_train, dataset_aug]))`
